microcosm_flask/namespaces.py

```diff
@@ -95,30 +98,30 @@ def endpoint_for(self, operation):
         Examples: `foo.search`, `bar.search_for.baz`
 
         """
-        if self.object_ is not None:
-            return operation.value.pattern.format(
-                self.subject_name,
-                operation.value.name,
-                self.object_name,
-            )
-        else:
-            return operation.value.pattern.format(
-                self.subject_name,
-                operation.value.name,
-            )
+        return operation.value.pattern.format(
```

Comment:
I switched to keyword-based format strings so I can substitute without having to know which pattern is being used.